yt/frontends/boxlib/data_structures.py

```diff
@@ -743,19 +743,7 @@ def _parse_cparams(self):
                 param, vals = (s.strip() for s in line.split("="))
             except ValueError:
                 continue
-            if param == "amr.n_cell":
-                vals = self.domain_dimensions = np.array(vals.split(), dtype="int32")
-
-                # For 1D and 2D simulations in BoxLib usually only the relevant
-                # dimensions have a specified number of zones, but yt requires
-                # domain_dimensions to have three elements, with 1 in the additional
-                # slots if we're not in 3D, so append them as necessary.
-
-                if self.dimensionality == 1:
-                    vals = self.domain_dimensions = np.array([vals[0], 1, 1])
-                elif self.dimensionality == 2:
-                    vals = self.domain_dimensions = np.array([vals[0], vals[1], 1])
-            elif param == "amr.ref_ratio":
+            if param == "amr.ref_ratio":
                 vals = self.refine_by = int(vals[0])
             elif param == "Prob.lo_bc":
```

Comment:
does this work when we specify several different ref_ratios for different jumps between levels?  e.g.

```
amr.ref_ratio = 2 4 2 2 
```

Reply:
It "works" in the sense that it will not crash if you give it vector - it takes the first value for the ref_ratio between levels 0 and 1. But I don't think yt itself supports different ref_ratios between different levels.

Reply:
I think (hope) it does, since we use it with that all the time.

Reply:
That's right, it's not supported _yet_.
Maybe you'd want to have a warning here in this case, because it's not necessarily obvious from the user standpoint

Reply:
In any case, this PR doesn't change anything about the way `ref_ratio` is handled, so perhaps this could be sorted out in a different issue / pull request?

Reply:
agree.  That just caught my eye.